lesson07.blog-api-server/api.js

```diff
@@ -0,0 +1,23 @@
+module.exports.getData = getData;
+
+function getData(dataFile, filters = []) {
+    if (dataFile) {
+        return (filterArray(JSON.parse(dataFile), filters));
+    }
+    return 'requested data currently unavailable. please try again in a few minutes';
```

Comment:
Not very informative - I pressed the button with empty input and got this message.